src/components/Mobile/components/SearchMobile/SearchMobile.tsx

```diff
@@ -0,0 +1,102 @@
+import { useRef, useState } from "react";
+import SlidingPane from "react-sliding-pane";
+import "react-sliding-pane/dist/react-sliding-pane.css";
+import { Input } from "../../../Search/style";
+import backIcon from "../../../../assets/icons/back.svg";
+import useInput from "../../../../hooks/useInput";
+import useLocalStorage from "../../../../hooks/useLocalStorage";
+import { isNotEmpty } from "../../../../utils/utils";
+import { SearchForm, Icon } from "./style";
+import SearchDropdown from "../../../SearchDropdown/SearchDropdown";
+import deleteIcon from "../../../../assets/icons/exit.svg";
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "../../../../store";
+import { filtersActions } from "../../../../store/slicers/filtersSlice";
+
+export interface SearchMobileProps {
+  isPaneOpen: boolean;
+  onClose: (value: boolean) => void;
+}
+const SearchMobile = (props: SearchMobileProps) => {
```

Comment:
if you want to improve it, consider union all the logic functions in one hook to save code on mobile and desktop